deep-learning/_posts/2017-09-17-3dbiomed-segmen.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title:  "Joint Sequence Learning and Cross-Modality Convolution for 3D Biomedical Segmentation"
+tags:   3D Biomedical Segmentation
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1704.07754.pdf   
+cite:
+  authors: "Kuan-Lun Tseng, Yen-Liang Lin, Winston Hsu, Chung-Yang Huang1"
+  title:   "Joint Sequence Learning and Cross-Modality Convolution for 3D Biomedical Segmentation"
+  venue:   "CVPR 2017"
+---
+
+### Description
+This paper proposes an end-to-end deep encoder-decoder network for 3D biomedical segmentation. The network is a combination of three main parts: multi-modal encoder, cross-modality convolution, and convolutional LSTM. 
+
+Four different modalities of (MRI) image are commonly referenced for the brain tumor surgery: T1, T1C, T2, and FLAIR. As shown in below, the slices from four different modalities are stacked together by the depth values. Then, they pass through different CNNs in the multi-modal encoder (each CNN is applied to a different modality) to obtain a semantic latent feature representation. Latent features from multiple modalities are effectively aggregated by the proposed cross-modality convolution layer. Then, convolutional LSTM  exploits the spatial and sequential correlations of consecutive slices. Finally, a 3D image segmentation is generated by concatenating a sequence of 2D prediction results. This model tries jointly exploit the correlations between different modalities and the spatial and sequential dependencies for consecutive slices. 
```

Comment:
as shown below

by the depth values -> along the axial orientation

to jointly exploit